doc/man5/x509v3_config.pod

```diff
@@ -284,10 +284,18 @@ B<access_id;location>, where B<access_id> is an object identifier
 (although only a few values are well-known) and B<location> has the same
 syntax as subject alternative name (except that B<email:copy> is not supported).
 
+Possible values for access_id are B<OCSP> (OCSP responder),
```

Comment:
```suggestion
Possible values for access_id include B<OCSP> (OCSP responder),
```

The implementation and spec are both pretty generic about the accessMethod of the AccessDescription, so I'd be a little reluctant to imply that this list is exhaustive (which is what the "are" seems to do, for me).